nixos/modules/services/desktop-managers/plasma6.nix

```diff
@@ -302,7 +302,10 @@ in
     services.displayManager.sddm = {
       package = kdePackages.sddm;
       theme = mkDefault "breeze";
-      wayland.compositor = "kwin";
+      wayland = {
+        enable = true;
+        compositor = "kwin";
```

Comment:
Should these maybe be behind a `mkDefault`?

Reply:
Good call.